qcfractal/queue/managers.py

```diff
@@ -321,20 +321,26 @@ def update(self, new_tasks: bool=True) -> bool:
         self.assert_connected()
 
         results = self.queue_adapter.acquire_complete()
-        if len(results):
+        n_success = 0
+        n_fail = 0
+        n_result = len(results)
+        if n_result:
             payload = self._payload_template()
 
             # Upload new results
             payload["data"] = results
             try:
-                response = self.client._automodel_request("queue_manager", "post", payload)
+                _ = self.client._automodel_request("queue_manager", "post", payload)
             except IOError:
                 # TODO something as we didnt successfully add the data
                 self.logger.warning("Post complete tasks was not successful. Data may be lost.")
 
-            self.active -= len(results)
+            self.active -= n_result
+            n_success = [r.success for r in results.values()].count(True)
```

Comment:
`sum([r.success for r in results.values()])`, but thats the NumPy in me.

Reply:
That is what I was originally going to do as well, but from what I was reading this is actually just a bit faster:

https://stackoverflow.com/a/12766783/10364409

Reply:
For items like this speed isn't an issue so I would go with which is more clear. No real preference.